pkgs/tools/security/spectre-meltdown-checker/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, makeWrapper, binutils-unwrapped }:
+
+stdenv.mkDerivation rec {
+  name = "spectre-meltdown-checker-${version}";
+  version = "0.29";
+
+  src = fetchFromGitHub {
+    owner = "speed47";
+    repo = "spectre-meltdown-checker";
+    rev = "v${version}";
+    sha256 = "14i9gx1ngs3ixjirlx4qd87pmac916rvv9y61a5f7nl0dig4awl4";
+  };
+
+  prePatch = ''
+    substituteInPlace spectre-meltdown-checker.sh \
+      --replace /bin/echo echo \
```

Comment:
there is a trailing `\` here. Probably leftover from the previous version.

Reply:
Done